src/cogs/bert.py

```diff
@@ -128,6 +147,24 @@ async def norbert(self, ctx):
         if ctx.invoked_subcommand is None:
             await ctx.send_help(ctx.command)
 
+    @norbert.command(name="top")
+    async def top(self, ctx, *content):
```

Comment:
add multi-line comment to method. these comments work as command descriptors and will show up in the help commands